pubsub/index.js

```diff
@@ -7,14 +7,18 @@
 const async = require('async')
 const crypto = require('crypto')
 const is = require('check-types')
+const logger = require('pino')({
+  // https://cloud.google.com/logging/docs/agent/configuration#timestamp-processing
+  timestamp: () => `,"time":"${(Date.now() / 1000.0).toFixed(3)}000000"`
```

Comment:
I had to squint at this for a bit! Curious why we truncate at 3 digits, then append 6 zeros to the timestamp string, instead of just doing `toFixed(9)`.